uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/components/manager-leave-calendar.js

```diff
@@ -304,15 +308,14 @@ define([
      * @return {Promise}
      */
     function loadCalendar() {
-      var promises = [];
-      _.each(vm.managedContacts, function (contact, index) {
-        promises.push(Calendar.get(contact.id, vm.selectedPeriod.id)
-          .then(function (calendar) {
-            vm.managedContacts[index].calendarData = setCalendarProps(vm.managedContacts[index].id, calendar);
-          }));
-      });
+      tempContactData = angular.copy(vm.managedContacts);
```

Comment:
you already have `_` at your disposal, do you need to inject `angular` also? Isn't `_.clone()` enough?